### PR TITLE
Expose edit_selected in Tree

### PR DESCRIPTION
**Pull request by Jummit**
Proposed in https://github.com/godotengine/godot-proposals/issues/2088#issuecomment-756915749.

Should be an easy cherry pick for 3.2.

### CODE REVIEW

File: doc/classes/Tree.xml [hunk 1]
```diff
@@ -123,6 +123,11 @@
 				Returns the column for the currently edited item.
 			</description>
 		</method>
+		<method name="edit_selected">
+			<description>
+				Edits the selected tree item as if it was clicked. The item must be set editable with [method TreeItem.set_editable]. Returns true if the item could be edited. Fails if no item is selected.
```
Comment (akien-mga) [inline doc/classes/Tree.xml:128]: The `<return ...>` type information seems to be missing. Make sure to use `godot --doctool .` to generate the template, not adding the XML manually. See https://docs.godotengine.org/en/stable/community/contributing/updating_the_class_reference.html

```suggestion
				Edits the selected tree item as if it was clicked. The item must be set editable with [method TreeItem.set_editable]. Returns [code]true[/code] if the item could be edited. Fails if no item is selected.
```

Comment (Jummit) [reply doc/classes/Tree.xml]: Done.

**Review by akien-mga — APPROVED** [review]

**Comment by akien-mga** [comment]
Thanks!

**Comment by akien-mga** [comment]
Cherry-picked for 3.2.4.

**Pull request merged**